include/sys/spa.h

```diff
@@ -219,7 +220,7 @@ typedef struct zio_cksum_salt {
  * G		gang block indicator
  * B		byteorder (endianness)
  * D		dedup
- * X		encryption (on version 30, which is not supported)
+ * X		encryption
```

Comment:
Shoudl I add a 4th ASCII art diagram showing this? or would it be sufficient to talk about this in the encrypted bp section.